lib/modules/platform/gerrit/utils.ts

```diff
@@ -0,0 +1,114 @@
+import { CONFIG_GIT_URL_UNAVAILABLE } from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import type { BranchStatus, PrState } from '../../../types';
+import * as hostRules from '../../../util/host-rules';
+import { parseUrl } from '../../../util/url';
+import { hashBody } from '../pr-body';
+import type { Pr } from '../types';
+import type {
+  GerritChange,
+  GerritChangeStatus,
+  GerritLabelTypeInfo,
+} from './types';
+import { TAG_PULL_REQUEST_BODY } from './types';
+
+export function getGerritRepoUrl(repository: string, endpoint: string): string {
+  // Find options for current host and determine Git endpoint
+  const opts = hostRules.find({
+    hostType: 'gerrit',
+    url: endpoint,
+  });
+
+  const url = parseUrl(endpoint);
+  if (!url) {
+    throw new Error(CONFIG_GIT_URL_UNAVAILABLE);
+  }
+  url.protocol = url.protocol.slice(0, -1);
```

Comment:
```suggestion
  url.protocol = url.protocol;
```
why slice?

Reply:
not sure, makes no sense to me.
this function was copied from another user without a closer look onto it.
i will remove it.